test/cypress/cypress/integration/features/management/rules/draft--check-rows-dropdow.feature

```diff
@@ -0,0 +1,51 @@
+Feature: Validate row dropdown on Rule page
+
+    As a wazuh user
+    i want to see the Rules pages
```

Comment:
In English, the first-person pronoun is always capitalized (I).

Reply:
Done